.github/workflows/unit-test.yml

```diff
@@ -35,8 +35,15 @@ jobs:
       fail-fast: false  # We still want to see all jobs' build result to the end
       matrix:
         module:
+          - generator
           - core
           - clients
```

Comment:
it's better that we maintain the same order for these items in all three files.

Reply:
Good point. Made it lexicographical everywhere and added comments, so that people adding more components in the future abide by it.